config/locales/zh.yml

```diff
@@ -581,7 +581,7 @@ doc_auth.headings.verify_at_post_office: 去邮局验证身份
 doc_auth.headings.verify_identity: 验证你的身份
 doc_auth.headings.verify_online: 在网上验证身份
 doc_auth.headings.verify_online_selfie: 使用手机在网上验证你的身份
-doc_auth.headings.welcome: 开始验证你的身份
+doc_auth.headings.welcome: 现在我们来为%{sp_name}验证你的身份
```

Comment:
Should this be removed from `ALLOWED_INTERPOLATION_MISMATCH_LOCALE_KEYS` ? 

Do we not have something which checks for unnecessary values in that array?

Reply:
Hm, yes. In looking into it, it doesn't seem to do that check (though it's intended), will open a separate PR with a fix